mobile/tools/what_to_run.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+
+BRANCH_NAME="$GITHUB_REF_NAME"
+BASE_COMMIT="$(git merge-base origin/main HEAD)"
+CHANGED_FILES="$(git diff "${BASE_COMMIT}" --name-only)"
+CHANGE_MATCH='^mobile/|^bazel/repository_locations\.bzl|^\.bazelrc|^\.github/workflows/mobile-*|^\.github/workflows/env.yml'
+
+
+run_default_ci () {
+    {
+        echo "mobile_android_build=true"
```

Comment:
do we still skip these on docs only PRs etc?

Reply:
i believe the conditions are the same - /docs is not in the `CHANGE_MATCH` so it would be skipped